inputs/orchestrator_inner:2.json

```diff
@@ -0,0 +1,48 @@
+{
+  "client_version": "{{VERSION}}",
+  "prebuild_plugins": [
+    {
+      "name": "pull_base_image",
+      "args": {
+        "parent_registry": "{{REGISTRY_URI}}",
+        "parent_registry_insecure": true
+      }
+    },
+    {
+      "name": "bump_release",
```

Comment:
The following has been added to `orchestrator_inner:1.json` recently:
```json
    {
      "name": "add_labels_in_dockerfile",
      "args": {
        "labels": "{{IMPLICIT_LABELS}}"
      }
    }
```
Let's add it here too (right before `bump_release`).